bedrock/firefox/templates/firefox/browsers/mobile/index.html

```diff
@@ -117,11 +117,12 @@ <h2 class="c-landing-grid-title"><a href="{{ url('firefox.browsers.mobile.ios')
 
     <div class="c-landing-grid-item">
       {{ resp_img(
-        url='img/firefox/browsers/mobile/index/firefox-focus.jpg',
+        url='img/firefox/mobile/firefox-focus.jpg',
         srcset={
-          'img/firefox/browsers/mobile/index/firefox-focus-high-res.jpg': '2x'
+          'img/firefox/mobile/firefox-focus-high-res.jpg': '2x'
```

Comment:
I have left the default imgs in their original location too for now, but wondering what's the best practice? Remove them and keep only in `l10/en-US`? (I'd ideally symlink them, from `img/l10n/en-US/*` to `img/firefox/browsers/*`, but not sure how the tooling likes that…)

Reply:
Fwiw, pages that switch out images best on language should have said images in the `/img/l10n` so the back-end logic switches them out, so you correctly placed them here 👍🏼